Public/Src/Engine/UnitTests/Processes/SandboxedProcessTestBase.cs

```diff
@@ -35,7 +37,15 @@ public SandboxedProcessTestBase(ITestOutputHelper output) : base(output)
             var envVars = Override(
                 BuildParameters.GetFactory().PopulateFromEnvironment().ToDictionary(),
                 overrideEnvVars);
-            
+
+            var testMethodFrame = new System.Diagnostics.StackTrace()

```

Comment:
>testMethodFrame [](start = 16, length = 15)

Maybe this should go into the BuildXLTest baseclass?

Reply:
I'll check and move it there if possible

Reply:
extracted `DiscoverCurrentlyExecutingXunitTestMethodFQN()` method and used it in other places where `SandboxedProcessInfo` is created